### PR TITLE
feat: added cli command to send usd or sats explicitly

### PR DESCRIPTION
**Pull request by sandipndev**
Allows for things like:

```bash
galoy-cli pay --username test --wallet btc --sats 50
galoy-cli pay --username test --wallet usd --cents 25
```

- Adds intraLedgerUsdPaymentSend

### CODE REVIEW

File: src/client/mod.rs [hunk 1]
```diff
@@ -207,6 +207,51 @@ impl GaloyClient {
...
+    pub fn intraleger_usd_send(
+        &self,
+        username: String,
+        amount: Decimal,
+        memo: Option<String>,
+    ) -> anyhow::Result<UsdPaymentSendResult> {
+        let me = self.me()?;
+        let wallet_id = me.default_account.default_wallet_id;
+
+        let recipient_wallet_id = self.default_wallet(username)?;
```
Comment (eshanvaid) [inline src/client/mod.rs:219]: What if the default wallets of the sender and receiver are different - one USD and other BTC?  Will `IntraLedgerUsdPaymentSend` work in that case?
Last time I tried this, it led to a `MismatchCurrency` error.

File: src/client/mod.rs [hunk 2]
```diff
@@ -207,6 +207,51 @@ impl GaloyClient {
...
     }
 
+    pub fn intraleger_usd_send(
+        &self,
+        username: String,
+        amount: Decimal,
+        memo: Option<String>,
+    ) -> anyhow::Result<UsdPaymentSendResult> {
+        let me = self.me()?;
+        let wallet_id = me.default_account.default_wallet_id;
```
Comment (eshanvaid) [inline src/client/mod.rs:217]:
```suggestion
        let sender_wallet_id = me.default_account.default_wallet_id;
```

File: src/client/mod.rs [hunk 3]
```diff
@@ -207,6 +207,51 @@ impl GaloyClient {
...
+
+        let recipient_wallet_id = self.default_wallet(username)?;
+        let input = IntraLedgerUsdPaymentSendInput {
+            amount,
+            memo,
+            recipient_wallet_id,
+            wallet_id,
```
Comment (eshanvaid) [inline src/client/mod.rs:224]:
```suggestion
            sender_wallet_id,
```

File: src/client/mod.rs (hunk 1 above)
Comment (sandipndev) [reply src/client/mod.rs:219]: Yes, it's supposed to work. Will test on staging.

Comment (sandipndev) [reply src/client/mod.rs:235]: Fixed

**Pull request merged**